src/main/java/org/opensearch/geospatial/ip2geo/action/UpdateDatasourceRequest.java

```diff
@@ -36,7 +36,6 @@
 public class UpdateDatasourceRequest extends ActionRequest {
     public static final ParseField ENDPOINT_FIELD = new ParseField("endpoint");
     public static final ParseField UPDATE_INTERVAL_IN_DAYS_FIELD = new ParseField("update_interval_in_days");
-    private static final int MAX_DATASOURCE_NAME_BYTES = 255;
```

Comment:
Where did we use this constant ?

Reply:
No where. I guess, it was left there by mistake.